Header header

int32 id
geometry_msgs/Polygon polygon
